### PR TITLE
[RELEASE-0.20] Add support for custom manifests

### PR DESCRIPTION
**Pull request by skonto**
Backport of #468, need it downstream.
/cc @matzew

### CODE REVIEW

File: pkg/reconciler/knativeeventing/knativeeventing.go [hunk 1]
```diff
@@ -106,7 +106,7 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, ke *v1alpha1.KnativeEven
 		common.CheckDeployments,
 		common.DeleteObsoleteResources(ctx, ke, r.installed),
 	}
-	manifest := r.manifest.Append()
+	manifest := r.manifest.Append(r.extension.Manifests()...)
```
Comment (matzew) [inline pkg/reconciler/knativeeventing/knativeeventing.go:109]: why is this different than here:

https://github.com/knative/operator/pull/468/files#diff-fc7141af4d915a54d757eac679ca6294d9d25d2493f12c8ad910afdb6678b906R104-R143

Comment (skonto) [reply pkg/reconciler/knativeeventing/knativeeventing.go:109]: Something is wrong with the cherry picking. Let me check.

Comment (skonto) [reply pkg/reconciler/knativeeventing/knativeeventing.go]: @matzew now its ok. Sorry for the messed up patch.

**Review by matzew — APPROVED** [review]
/lgtm
/approve

**Pull request merged**